include/nccl_ofi_rdma.h

```diff
@@ -352,7 +352,9 @@ typedef struct {
 	nccl_net_ofi_rdma_mr_handle_t *mr_handle;
 	/* Schedule used to transfer this request. We save the pointer to
 	 * reference it when transferring the request over network. */
-	nccl_net_ofi_schedule_t *schedule;
+	nccl_net_ofi_schedule_t *schedule[MAX_NUM_RAILS];
```

Comment:
I don't think there's any advantage to using the scheduler here.  Just iterate over the endpoint rails.

Reply:
ack